ddp-workspace/projects/ddp-singular/src/app/components/pages/home/home.component.html

```diff
@@ -111,23 +111,23 @@ <h1>Participating is Easy</h1>
     <h1>Our Partners</h1>
     <div class="partners-list">
       <div class="partners-list-item">
-        <a target="_blank" [href]="ADDITIONAL_VENTURES_URL">
-          <img src="assets/images/partner-1.png" alt="Additional Ventures" />
+        <a href="https://www.additionalventures.org/" target="_blank">
```

Comment:
I believe related [ticket](https://broadinstitute.atlassian.net/browse/DDP-7315) is already done and tested. So what's the purpose of these changes?

Reply:
Fixed this moment, please check it again.